src/_common/SizeProvider.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default class SizeProvider extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            height: '600px',
+            width: '1000px',
+        };
+    }
+
+    static propTypes = {
+        windowSizeRelation: React.PropTypes.func.isRequired,
```

Comment:
a function that compute width and height with respect of window size, is this what you want?